js/stores/windowStore.js

```diff
@@ -36,7 +36,7 @@ let windowState = Immutable.fromJS({
 let lastEmittedState
 
 const CHANGE_EVENT = 'change'
-const PDFJS_ORIGIN = 'chrome-extension://adnmjfhcejodgpaljdmlmjoclihpcfka/'
```

Comment:
This is the main bug that here it is using `adnmjfhcejodgpaljdmlmjoclihpcfka` but above it installs as `oemmndcbldboiebfnladdacbdfmadadm`